.travis.yml

```diff
@@ -15,15 +15,15 @@ matrix:
     - python: '2.7'
       env:
         - TOXENV=du13
-        - PYTEST_ADDOPTS = --cov sphinx --cov-append --cov-config setup.cfg
+        - PYTEST_ADDOPTS="--cov ./ --cov-append --cov-config setup.cfg"
```

Comment:
Unfortunately, this is not recognized as envvar by travis. So it was not defined on runtime.